lib/glific/flags.ex

```diff
@@ -38,7 +35,9 @@ defmodule Glific.Flags do
   end
 
   defp out_of_office_check do
-    {:ok, now} = DateTime.now(@timezone)
+    timezone = Glific.Partners.organization().timezone
+
+    {:ok, now} = DateTime.now(timezone)
```

Comment:
Since timezone is a user input it's possible that this have a invalid time on so can you handle that case as well?